service-base/pom.xml

```diff
@@ -120,6 +120,10 @@
       <groupId>io.opentracing.contrib</groupId>
       <artifactId>opentracing-tracerresolver</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+    </dependency>
```

Comment:
this dependency is not needed when I compile and run integration tests using Java 10